src/Kunstmaan/AdminBundle/Resources/config/services.yml

```diff
@@ -88,6 +88,10 @@ services:
         class: Kunstmaan\AdminBundle\Helper\CloneHelper
         arguments: ['@doctrine.orm.entity_manager', '@event_dispatcher']
 
+    kunstmaan_admin.isadminroute.helper:
+        class: Kunstmaan\AdminBundle\Helper\IsAdminRouteHelper
+        arguments: ['%admin_url%']
```

Comment:
@aistis- 
I've been puzzled about the best way to attack how I should actually make this parameter configurable. For now I settled on just relying on the application parameters.yml. Bundle configuration did not seem like the right tool for the job because of a few reasons. 

Configuration for this parameter should be duplicated across all the bundles included in this PR or advanced code in the AdminBundle is needed to check if any of the other depended bundles are active and pass the parameter on. Having this parameter defined for each bundle also makes the configuration a lot more error prone, one could override the admin_prefix for one bundle but fail to adjust it in the adminbundle.

A custom route loader would be necessary I believe to allow for this amount of configuration without using the symfony parameters.yml. If possible I would like to avoid that and keep it simple.

It also makes sense to put this configuration option in the application parameters.yml in my opinion. It's never a choice you make for each bundle on its own but rather for the whole application.

If you have other opinions on this topic or have more deep knowledge on how to workaround this issue, please let me know. That is exactly the reason why I made this PR public so soon before finishing it. 

@dannyvw I was thinking about putting admin_prefix: 'admin' as a default in the parameters.yml.dist of our kunstmaanbundlesstandardedition. That way new projects would have the option set by default.